autoload/youcompleteme/hierarchy.vim

```diff
@@ -85,6 +86,8 @@ function! s:MenuFilter( winid, key )
     endif
     call win_execute( s:popup_id,
                     \ 'call cursor( [' . string( s:select ) . ', 1 ] )' )
+    call win_execute( s:popup_id,
+                    \ 'set cursorline cursorlineopt&' )
```

Comment:
Just to make sure I understand everything, this is the hack you mentioned in the pull request description, right?

You mentioned `cursorline`, but did you need `cursorlineopt&` as well?

Reply:
I vaguely recall this being necessary from the finder popup. I copied this line from there. I don’t recall why though.